server/src/main/java/com/sauron/model/entities/User.java

```diff
@@ -0,0 +1,38 @@
+package com.sauron.model.entities;
+
+import javax.persistence.*;
+
+@Entity
+@Table(name = "USERS")
+public class User {
```

Comment:
User entity will be used for authentication so password field is required. I think that username field can be deleted and we can use email instead. If you will leave both, I suggest marking username as unique, non-updatable and non-nullable.

Reply:
Ok for constraints. 
However, we don't know so far how we will implement authorization, so I would ignore password so far (we could enable authorization by social media for example and then password could be unnecessary).